src/main/java/com/hpe/simulap/protocol/sip/config/SipNodeContext.java

```diff
@@ -88,7 +89,8 @@
 public class SipNodeContext implements SipListener {
 
 	private SipNodeElement sipNode = null;
-	private static final Logger _logger = LoggingManager.getLoggerForClass();
+
+	private static final Logger _logger = LoggerFactory.getLogger(SipNodeContext.class);
```

Comment:
Why changing the logger unless using its features for any info, warn, debug methods?